src/libcollections/borrow.rs

```diff
@@ -249,6 +249,7 @@ impl<'a, B: ?Sized> Default for Cow<'a, B>
     where B: ToOwned,
           <B as ToOwned>::Owned: Default
 {
+    /// Creates a `Cow<'a, B>` pointer.
```

Comment:
This is wrong. `Cow` is not a pointer either. `Default` for `Cow` is

> Creates an owned `Cow<'a, B>` with the default value for the contained owned value.

or something.